examples/n-body/README.md

```diff
@@ -31,17 +31,17 @@ Benchmark
 
 ***Environment:***
 - MacBook Pro (15-inch, 2019)
-- macOS 10.14.6
-- node.js v13.5.0
+- macOS 10.15.3
+- node.js v13.8.0
 - rustc 1.42.0-nightly (3a3f4a7cb 2019-12-28)
 
 ***Results:***
 
 |        Target           |  Time, ***ms*** | Size, ***KB*** |
 |-------------------------|-----------------|----------------|
-| **AssemblyScript WASM** | **1650**        | **1.6**        |
-| AssemblyScript ASMJS    | 2558            | 10*            |
-| JavaScript              | 1596            | 5*             |
-| Rust WASM               | 1570            | 2              |
+| **AssemblyScript WASM** | **1602**        | **1.6**        |
+| AssemblyScript ASMJS    | 2368            | 10*            |
+| JavaScript              | 1616            | 5*             |
+| Rust WASM               | 1618            | 2              |
```

Comment:
On my machine it sometimes looks like this

```
COLD SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1490.901901ms
Performing 20000000 steps (AssemblyScript ASMJS) ...
Took 2229.8403ms
Performing 20000000 steps (JS) ...
Took 1505.058601ms
Performing 20000000 steps (Rust WASM) ...
Took 1493.7861990000001ms

WARMED UP SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1486.542901ms
Performing 20000000 steps (AssemblyScript ASMJS) ...
Took 2227.368101ms
Performing 20000000 steps (JS) ...
Took 1497.3253ms
Performing 20000000 steps (Rust WASM) ...
Took 1495.026099ms
```

and sometimes like this

```COLD SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1500.644099ms
Performing 20000000 steps (AssemblyScript ASMJS) ...
Took 2254.637401ms
Performing 20000000 steps (JS) ...
Took 1498.8344ms
Performing 20000000 steps (Rust WASM) ...
Took 1487.467101ms

WARMED UP SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1488.9312ms
Performing 20000000 steps (AssemblyScript ASMJS) ...
Took 2314.464401ms
Performing 20000000 steps (JS) ...
Took 1492.047499ms
Performing 20000000 steps (Rust WASM) ...
Took 1482.7864ms
```

indicating that there's not much of a difference in AS/Rust/JS perf, but varies slightly from run to run. Makes me wonder if we should extend the benchmark script to take the mean of multiple runs for each?

Reply:
Variance approx 0.8-1% so I don't think necessary build more complex bench

Reply:
Yeah, it's not much, yet feels like one of those occasions where being extra careful is exactly what we'd wish others did as well when it comes to numbers, so it would be great to lead by example :)

Reply:
How about just add notice about: "results may vary from run to run?"